Sources/Testing/Running/Runner.swift

```diff
@@ -114,6 +114,51 @@ extension Runner {
     }
   }
 
+  /// Execute the ``CustomExecutionTrait/execute(_:for:testCase:)`` functions
+  /// associated with the test in a plan step.
+  ///
+  /// - Parameters:
+  ///   - step: The step being performed.
+  ///   - testCase: The test case, if applicable, for which to execute the
```

Comment:
Nitpick: the whitespace in the documentation for this symbol does not match house style. (Gotta love comments about whitespace.)

Reply:
Nah you're good, I very much like being consistent, sorry for not comparing against the other styles myself!